explainer.md

```diff
@@ -1,8 +1,9 @@
 # Overview
 
 Author: Matt Giuca &lt;<mgiuca@chromium.org>&gt;
+Author: Eric Willigers &lt;<ericwilligers@chromium.org>&gt;
 
-Date: 2017-09-22
+Date: 2018-09-04
```

Comment:
Can you change this to "Created: ... / Updated: ..." so it doesn't look like the document was just written.